test/acceptance/integration/cb_mdw/cb_mdw.feature

```diff
@@ -279,11 +294,12 @@ Feature: Context broker middleware
     | json   | /v1/registry/contextAvailabilitySubscriptions/subscriptionId | DELETE |
 
   Scenario Outline: Parameters-Query in cb urls with operation in the payload
-    Given a domain in KEYSTONE
-    And a user in the domain
-    And a project in the user
-    And url with "/v1/updateContext?details=on&limit=15&offset=0" and the actionType attribute "<attribute>"
-    When a context broker "POST" petition is asked to PEP with "<format>" format
+    Given a KEYSTONE CONFIGURATION with all roles in the same project
+    And set the request URL with the path "/v1/updateContext?details=on&limit=15&offset=0"
```

Comment:
The `/v1/updateContext?details=on&limit=15&offset=0` is supposed to map a real NGSI operation? Or is it only for testing somekind of syntactic structure? In the first case, details/limit/offset make sense for query-like operations (e.g. queryContext), in the case of updateContext is a bit weird...